pkg/jobconfig/files.go

```diff
@@ -25,21 +27,24 @@ import (
 )
 
 type label string
+type generator string
 
 const (
-	CanBeRehearsedLabel          = "pj-rehearse.openshift.io/can-be-rehearsed"
-	CanBeRehearsedValue          = "true"
-	SSHBastionLabel              = "dptp.openshift.io/ssh-bastion"
-	ProwJobLabelVariant          = "ci-operator.openshift.io/variant"
-	ReleaseControllerLabel       = "ci-operator.openshift.io/release-controller"
-	LabelGenerated               = "ci-operator.openshift.io/prowgen-controlled"
-	ReleaseControllerValue       = "true"
-	JobReleaseKey                = "job-release"
-	PresubmitPrefix              = "pull"
-	PostsubmitPrefix             = "branch"
-	PeriodicPrefix               = "periodic"
-	Generated              label = "true"
-	NewlyGenerated         label = "newly-generated"
+	CanBeRehearsedLabel              = "pj-rehearse.openshift.io/can-be-rehearsed"
+	CanBeRehearsedValue              = "true"
+	SSHBastionLabel                  = "dptp.openshift.io/ssh-bastion"
+	ProwJobLabelVariant              = "ci-operator.openshift.io/variant"
+	ReleaseControllerLabel           = "ci-operator.openshift.io/release-controller"
+	LabelCluster                     = "ci.openshift.io/cluster"
+	LabelGenerator                   = "ci.openshift.io/generator"
+	ReleaseControllerValue           = "true"
+	JobReleaseKey                    = "job-release"
+	PresubmitPrefix                  = "pull"
+	PostsubmitPrefix                 = "branch"
+	PeriodicPrefix                   = "periodic"
+	NewlyGenerated         label     = "newly-generated"
+	Prowgen                generator = "prowgen"
+	ClusterInit            generator = "cluster-init"
```

Comment:
Nothing in here uses these constants, so they can be moved to their respective modules (`pkg/prowgen`, `cmd/cluster-init`).

Reply:
I was thinking that, but the `generator` type is defined here, so I wasn't sure what the best practice was with that. I'll move them.